From 5780f27451f4a5a0672e2582743f53e2807a8ced Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Wed, 13 Aug 2014 00:03:19 -0400
Subject: [PATCH] Change Authentication Servers Down Message

State it is not this servers fault, and direct them to a status page.
---
 src/main/java/net/minecraft/server/ThreadPlayerLookupUUID.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/net/minecraft/server/ThreadPlayerLookupUUID.java b/src/main/java/net/minecraft/server/ThreadPlayerLookupUUID.java
index 5086861..b72c085 100644
--- a/src/main/java/net/minecraft/server/ThreadPlayerLookupUUID.java
+++ b/src/main/java/net/minecraft/server/ThreadPlayerLookupUUID.java
@@ -52,7 +52,7 @@ class ThreadPlayerLookupUUID extends Thread {
                 LoginListener.a(this.a, this.a.a(gameprofile));
                 LoginListener.a(this.a, EnumProtocolState.READY_TO_ACCEPT);
             } else {
-                this.a.disconnect("Authentication servers are down. Please try again later, sorry!");
+                this.a.disconnect("Minecraft Authentication servers are down. Please try again later, sorry! \n\nThis is not a problem with our server. Please check http://xpaw.ru/mcstatus/"); // Spigot
                 LoginListener.e().error("Couldn\'t verify username because servers are unavailable");
             }
             // CraftBukkit start - catch all exceptions
-- 
1.9.1

